docs/api_reference.rst

```diff
@@ -182,14 +182,35 @@ specifications.
 Fonts
 ~~~~~
 
-WeasyPrint can use any font that Pango can find installed on the system. Fonts
-are automatically embedded in PDF files.
+WeasyPrint can use any font that Pango_ can find installed on the system. Fonts are
+automatically embedded in PDF files and are subset by default to only include the glyphs
+used in the PDF. Subsetting is done with Harfbuzz-Subset_ when available on the system,
+or by the slower fontTools_ library as a fallback.
 
-Pango always uses fontconfig to access fonts, even on Windows and macOS. You
+Pango_ always uses Fontconfig_ to access fonts, even on Windows and macOS. You
 can list the available fonts thanks to the ``fc-list`` command, and know which
 font is matched by a given pattern thanks to ``fc-match``. Copying a font file
 into the ``~/.local/share/fonts`` directory is generally enough to install a
-new font. WeasyPrint should support the major font formats handled by Harfbuzz.
+new font. WeasyPrint should support the major font formats handled by HarfBuzz_.
+
+WeasyPrint follows the Fontconfig_ configuration of the system. This default
+configuration is often useful for many cases: font fallbacks for missing glyphs, aliases
+for standard font families like "serif" or "monospace", colored variants for emojis,
+etc. But some of these default rules can sometimes interfere with CSS rules, and it may
+be interesting to disable them if you need to tweak details about font management.
+
+When a glyph is missing from a font and all its fallbacks, a `Notdef glyph`_ is
```

Comment:
In text shaping, a font’s public API is queried with Unicode code points, not glyph IDs. Therefore it’s really “a code point is not supported by fonts” instead of “a glyph is missing in fonts”.

```suggestion
When a Unicode code point is not supported by a font and its fallbacks, the font’s `.notdef glyph`_ is
```